ci.nix

```diff
@@ -24,10 +24,10 @@
     # cabal-install and nix-tools plans.  When removing a ghc version
     # from here (so that is no longer cached) also remove ./materialized/ghcXXX.
     # Update supported-ghc-versions.md to reflect any changes made here.
-    nixpkgs.lib.optionalAttrs (nixpkgsName == "R2009") {
+    nixpkgs.lib.optionalAttrs (nixpkgsName == "R2105") {
       ghc865 = false;
-      ghc8107 = false;
-    } // nixpkgs.lib.optionalAttrs (nixpkgsName == "R2105") {
+      ghc8107 = true;
```

Comment:
Should be `false`, I think? And change the corresponding line in the docs.

Reply:
Yes, exactly, I'm saying that we should change the value. I think that's our policy and I'm fine with it here. Note that we still *build* it, we just don't run as many tests.

Reply:
It isn't my resources being used for this, so I will defer to whatever you decide. However, from my point of view, I would prefer if there was some grace period (a month or two, at least), where both versions versions of nixos are fully tested.

I know we aren't ready to switch to nixos-21.11, and I'll be a little bit sad if my efforts to make sure our dependencies are ready to upgrade led to version we are actually using being less supported. (Though, to be honest, our more critical component is using GHC 8.6.5 still, so it wouldn't be changing anything in that regard.)

> I think that's our policy and I'm fine with it here.

Is this documented somewhere? I don't see anything [here](https://input-output-hk.github.io/haskell.nix/reference/supported-ghc-versions/) about expected time frames for when support levels for nixos versions will change (being dropped, stopping running tests).

Reply:
> Is this documented somewhere?

No, we should add something to the doc on supported versions!